content/departments/engineering/teams/growth/cloud-trial-operations.md

```diff
@@ -0,0 +1,106 @@
+  - this will send an alert in #cloud-trial-alerts and automatically send an email to the user to let them know that we are experiencing high demand and can't get them an instance at the moment.
+
+This workflow is very manual. We are working hard to automate all of this to improve the user experience massively, and reduce the operational burden. Come [join us](index.md)! All this workflow is powered by salesforce, and [Zapier automation](https://zapier.com/app/zaps/folder/1368159).
+
+### Instance provisioning and hand-off workflow
+
+@ebrodymoore and @malo will be monitoring the #cloud-trial-alerts for qualified requests. When a request is sent, they will:
+
+1. self-assign the request by replying "I'm taking it" to the alert
+1. raise a trial instance request with the cloud team (this step will be automated soon)
+1. when the instance is ready
+   1. set a license on the instance with 1,000 users, a 30-day limit, and the following tags: `plan:enterprise-0`,`acls`,`private-extension-registry`,`remote-extensions-allow-disallow`,`monitoring`,`true-up`,`batch-changes`,`code-insights`,`trial`
+   2. send the admin an onboarding email with
+      1. an initial password reset link to the instance
+      2. a link to the onboarding checklist
+      3. a link to schedule time with a customer engineer (CE)
+   3. set the lead status in salesforce to `Trial Instance Handed Off`
+
+### PQL qualification workflow
+
+The product growth team will be monitoring analytics for trial instances that have usage patterns that show signs that they could convert into a customer: Product Qualified Leads (PQL). For now, the criteria is best judgement and the workflow is manual, but this will be clearly defined and automated in the future.
+
+When a trial becomes a PQL, the lead status will be changed to `PQL` in salesforce, which will trigger an alert in #channel-TBD, to be picked up by an Account Executive.
+
+## Expectations from stakeholders
+
+Here's how every team at Sourcegraph contributes to the trial workflow in the short term, and in the long term.
+
+## Growth marketing and SDRs
+
+#### Now
+
+- Drive qualified traffic to the signup page
+- SDRs: see the [qualification workflow](qualification-workflow)
+
+# Product growth
+
+#### Now
+
+- After a cloud instance requests is qualfiied raise instance provisioning requests to the cloud team
+- When the instance is provisioned, hand it off to the user and make sure they can get started
+- 5 days max after the 30-day trial ends, make sure a decision is made to [extend, convert, or terminate](../../../../departments/cloud/trial_mi.md/#cloud-trial-managed-instances) the trial
+- monitor metrics for product qualified leads (PQLs)
+- (those are just operational tasks, the product growth team is also working on its usual produc team stuff)
+
+#### Long term
+
+- Automate it all!
+- Also see our [broader strategy](../../../../strategy-goals/strategy/growth-team/index.md).
+
+## Customer Engineering (CE)
```

Comment:
@amenne this is what we discussed yesterday. Does that work for you?

Reply:
CE side looks good. For Support I’d say our [normal SLAS](https://handbook.sourcegraph.com/departments/ce-support/support/#sts=SLAs) for now and we’ll revisit depending on quantity. (Before we have to go build and track a separate set).

Not sure where this fits but we talked about when a customer opts-in to 30 mins with CE we would generate a msg in #ce and tag the @ce-team. Right?

Reply:
Thanks.
@ngage1 can you build an automate that workflow? (with "chili something", if don't remember exactly)

Reply:
@seth-sourcegraph can you please include the CP link to schedule these 30 minute syncs- Dan and Casi can then loop in the CE.